src/areaSearch/actions.js

```diff
@@ -43,6 +43,7 @@ import type {
   ReceiveFileOperationFailedAction,
 } from '$src/areaSearch/types';
 import type {Attributes, Methods} from '$src/types';
+import type { SetAreaSearchAttachmentsAction } from "./types";
```

Comment:
`'` or `"`? I think we should use one, not both. This seems to apply in multiple files, please check